apm-lambda-extension/extension/route_handlers.go

```diff
@@ -68,13 +73,18 @@ func handleInfoRequest(handler *serverHandler, w http.ResponseWriter, r *http.Re
 
 // URL: http://server/intake/v2/events
 func handleIntakeV2Events(handler *serverHandler, w http.ResponseWriter, r *http.Request) {
-	bodyBytes, err := getDecompressedBytesFromRequest(r)
-	if nil != err {
-		log.Printf("could not get decompressed bytes from request body: %v", err)
+	if r.Body == nil {
+		log.Println("Could not get bytes from agent request body")
 	} else {
+		rawBytes, _ := ioutil.ReadAll(r.Body)
```

Comment:
It looks like `ioutil.ReadAll` doesn't promise success.  Should we be handling the error here instead of suppressing with an `_`? Or is that not needed?

Reply:
I'll ask what the apm server does, as we should probably handle the error similarly to how they would.

Reply:
https://github.com/elastic/apm-server/blob/master/beater/api/intake/handler.go#L72-L100

If the apm server can't read the request, it doesn't log the error but rather writes the error back to the client. I'm not sure we would want to write errors back to the agent, as the agent can't do anything more with the information than the extension can, and we don't want to hold up the lambda function. It seems to me like the best way to handle this would be to log the error and move on. @astorm what do you think? Maybe @felixbarny has some thoughts as well.

Reply:
SGTM. If we find use cases that require sending a response to the agent, we can always add it later.

Reply:
@estolfo logging the error and moving on seem sufficient to me as well.